page-data.php

```diff
@@ -279,17 +279,21 @@
           </div>
         </div>
         <!-- END OF TEMPLATE -->
-        <?php endforeach; ?>
+			<?php endforeach; ?>
       </div>
-      <div class="pagination_wrapper">
-        <?php if ($param_page > 1): ?>
-          <a href="<?php echo $prev_page_link; ?>"> << Previous Page </a>
-        <?php endif ?>
-        <?php if ($param_page < $total_pages): ?>
-          <a href="<?php echo $next_page_link; ?>"> Next Page >> </a>
-        <?php endif ?>
-      </div>
-    </div> <!-- end of right container -->
+
+			<section class="container">
```

Comment:
@DragonVirus you do not need to code the pagination of the **view all* page yourself, you can reuse this <section>. I will merge this PR once the indexing is finished and I can test it so do not worry about that, just mind that the pagination will set the **page** attribute on the URL (example: ?page=2 )